semanticdb/metacp/src/main/scala/scala/meta/internal/scalacp/SymbolInformationOps.scala

```diff
@@ -215,9 +215,29 @@ trait SymbolInformationOps { self: Scalacp =>
       }
     }
 
+    private val syntheticAnnotationsSymbols = Set(
+      "scala.reflect.macros.internal.macroImpl#"
```

Comment:
This needs to be changed to accommodate the recent changes in the symbol format. It should say `scala/reflect/macros/internal/macroImpl#`.